handlers/azure_queue_messages_api_v1/handler/init.rb

```diff
@@ -0,0 +1,164 @@
+# Require the dependencies file to load the vendor libraries
+require File.expand_path(File.join(File.dirname(__FILE__), "dependencies"))
+
+class AzureQueueMessagesApiV1
+  # ==== Parameters
+  # * +input+ - The String of Xml that was built by evaluating the node.xml handler template.
+  def initialize(input)
+    # Set the input document attribute
+    @input_document = REXML::Document.new(input)
+
+    # Retrieve all of the handler info values and store them in a hash variable named @info_values.
+    @info_values = {}
+    REXML::XPath.each(@input_document, "/handler/infos/info") do |item|
+      @info_values[item.attributes["name"]] = item.text.to_s.strip
+    end
+
+    # Retrieve all of the handler parameters and store them in a hash variable named @parameters.
+    @parameters = {}
+    REXML::XPath.each(@input_document, "/handler/parameters/parameter") do |item|
+      @parameters[item.attributes["name"]] = item.text.to_s.strip
+    end
+
+    @debug_logging_enabled = ["yes","true"].include?(@info_values['enable_debug_logging'].downcase)
+    @error_handling = @parameters["error_handling"]
+
+    @tenent_id = @info_values["tenent_id"]
+    @client_id = @info_values["client_id"]
+    @client_secret = @info_values["client_secret"]
+    
+    @headers = @parameters["headers"].to_s.empty? ? {} : JSON.parse(@parameters["headers"])
+    @body = @parameters["body"]
+    @method = (@parameters["method"] || :get).downcase.to_sym
+    @storage_account = @parameters["storage_account"]
+    puts @parameters
```

Comment:
Remove this. I assume you had it in here for troubleshooting during development.